FFXIVClientStructs/FFXIV/Client/Game/UI/Buddy.cs

```diff
@@ -45,11 +46,10 @@ public struct BuddyMember {
     [FieldOffset(0x1B58)] public BuddyMember* SquadronTrustPtr;
 
     public bool IsBuddyEquipUnlocked(uint buddyEquipId) {
-        fixed (byte* p = BuddyEquipUnlockBitmask) {
-            return IsBuddyEquipUnlockedInternal(p, buddyEquipId);
-        }
+        fixed (BuddyMember** ptr = &CompanionPtr)
+            return IsBuddyEquipUnlockedInternal(ptr, buddyEquipId);
     }
 
     [MemberFunction("E9 ?? ?? ?? ?? 0F B7 50 02 41 B8")]
-    private static partial bool IsBuddyEquipUnlockedInternal(void* ptr, uint buddyEquipId);
+    private static partial bool IsBuddyEquipUnlockedInternal(BuddyMember** companionPtr, uint buddyEquipId);
```

Comment:
Unsure if the generator spits this out as a public member still, but if so this should not be used in terms of plugins.

Reply:
what do you mean by "this should not be used in terms of plugins"? it's a private function, only accessable to `IsBuddyEquipUnlocked`. plugins can't use it

Reply:
Oh, I see, you mean this, right?

![FBHEpJ9SDH](https://github.com/aers/FFXIVClientStructs/assets/96642047/ee5805f8-cafc-47e8-a707-310566ab28d1)

Only the address is public, of course.